staging/src/k8s.io/apimachinery/pkg/api/resource/amount_test.go

```diff
@@ -131,3 +131,28 @@ func TestAmountSign(t *testing.T) {
 		}
 	}
 }
+
+func TestInt64AmountAsScaledInt64(t *testing.T) {
+	for _, test := range []struct {
+		value  int64
+		scale  Scale
+		scaled Scale
+		result int64
+		ok     bool
+	}{
+		{100, 0, 5, 1, true},
```

Comment:
Name the struct values.  Don't rely on the struct order.

```go
{
   value: 100,
   scale: 0,
```

Also, put a `name` in each test case and use the name in a `test.Run(test, func(t *testing.T){})` call

Reply:
@deads2k Done